### PR TITLE
chore(zh-cn): remove redundant seecompattable macro from some documents

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

remove {{seecompattable}} for some document and related content cleanup

### CODE REVIEW

File: files/zh-cn/web/api/element/replacechildren/index.md [hunk 1]
```diff
@@ -3,20 +3,21 @@ title: Element.replaceChildren()
...
+```js-nolint
+replaceChildren(param1)
+replaceChildren(param1, param2)
+replaceChildren(param1, param2, /* …, */ paramN)
 ```
 
 ### 参数
 
-- `nodesOrDOMStrings`
+- `param1`, …, `paramN`
```
Comment (jasonren0403) [inline files/zh-cn/web/api/element/replacechildren/index.md:20]:
```suggestion
- `param1`、…、`paramN`
```

File: files/zh-cn/web/api/element/replacewith/index.md [hunk 2]
```diff
@@ -3,45 +3,47 @@ title: Element.replaceWith()
...
+replaceWith(param1)
+replaceWith(param1, param2)
+replaceWith(param1, param2, /* …, */ paramN)
 ```
 
 ### 参数
 
-- `节点`
-  - : 一系列用来替换的{{domxref("Node")}} 对象或者 {{domxref("DOMString")}} 对象。
+- `param1`, …, `paramN`
```
Comment (jasonren0403) [inline files/zh-cn/web/api/element/replacewith/index.md:20]:
```suggestion
- `param1`、…、`paramN`
```

File: files/zh-cn/web/api/origin/index.md
Comment (jasonren0403) [inline files/zh-cn/web/api/origin/index.md:1]: 后续会提交PR更新该文档

File: files/zh-cn/web/api/element/replacewith/index.md [hunk 3]
```diff
@@ -3,45 +3,47 @@ title: Element.replaceWith()
...
-child.replaceWith(span);
+p.replaceWith(span);
 
-console.log(parent.outerHTML);
+console.log(div.outerHTML);
 // "<div><span></span></div>"
 ```
 
-### `ChildNode.replaceWith()` is unscopable
+### `replaceWith()` 是不可使用作用域的
```
Comment (yin1999) [inline files/zh-cn/web/api/element/replacewith/index.md:44]: check: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/glossary.md#javascript

```suggestion
### `replaceWith()` 是非作用域方法
```

**Review by yin1999 — APPROVED** [review]
Thanks @jasonren0403

**Pull request merged**